processing/src/main/java/io/druid/segment/IndexMerger.java

```diff
@@ -399,6 +416,35 @@ public String apply(@Nullable String input)
     return makeIndexFiles(indexes, outDir, progress, mergedDimensions, mergedMetrics, rowMergerFn, indexSpec);
   }
 
+  public static File reprocess(final File inDir, final File outDir, final IndexSpec indexSpec) throws IOException
```

Comment:
Eventually I think having "progress" be reportable by stuff other than the hadoop indexer would be handy. I'd rather leave it as is unless there's a compelling reason to get rid of it.